docker/daemon_unix.go

```diff
@@ -23,9 +23,6 @@ import (
 const defaultDaemonConfigFile = "/etc/docker/daemon.json"
 
 func setPlatformServerConfig(serverConfig *apiserver.Config, daemonCfg *daemon.Config) *apiserver.Config {
```

Comment:
I think at this point we can remove this function and it's call in `daemon/daemon.go` L222. We can also remove the windows version in `daemon/daemon_windows.go`.